olm-catalog/0.0.4/operator.nsx.vmware.com_ncpinstalls_crd.yaml

```diff
@@ -1 +1 @@
-../../deploy/operator.nsx.vmware.com_ncpinstalls_crd.yaml
\ No newline at end of file
+../../deploy/kubernetes/operator.nsx.vmware.com_ncpinstalls_crd.yaml
```

Comment:
It should be ../../deploy/openshift4/operator.nsx.vmware.com_ncpinstalls_crd.yaml